game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/TripleAMenuBar.java

```diff
@@ -30,7 +30,6 @@ public TripleAMenuBar(final TripleAFrame frame) {
       add(new NetworkMenu(watcher, frame));
     }
 
-    add(new WebHelpMenu());
```

Comment:
Looks like WebHelpMenu.java can be deleted.

Reply:
Done in latest commit.